nbs/examples/Exogenous.ipynb

```diff
@@ -7,58 +7,111 @@
    "source": [
```

Comment:
<p>Could we change the colab link? It directs to the cross validation nb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>